R/get_infect_mat.R

```diff
@@ -65,25 +65,28 @@ get_infect_mat <- function(alpha = 0.2, R = 2.5, kappa = 0.5, eta = 0.5, nu = 4,
     t_pa,
     offset = offset, shape = shape, rate = rate
   )
+
+  second_gen_shape <- shape + (shape / rate + t_incubation + offset)
+
   gamma_qcs <- get_prop_infect_time(
-    t_qcs - (t_incubation + (shape / rate) + offset),
-    shape = shape, rate = rate, offset = offset
+    t_qcs - (shape / rate),
```

Comment:
The value that is plugged into the `pgamma` function is the delay from symptom onset of the *index case* to quarantine of the contact minus the mean infectious time minus the offset. (The minus offset part is *built into* the `get_prop_infect_time()` function which is why you don't see it here)